Fastfile

```diff
@@ -53,24 +53,27 @@ platform :ios do
     match(type: "development", force_for_new_devices: true)
   end
 
-  desc "Submit a new Beta version to Crashlytics Beta"
-  lane :crashlytics_beta do
-    fabric_token = ENV["FABRIC_API_TOKEN"]
-    raise "Env var 'FABRIC_API_TOKEN' not set" unless fabric_token
-
-    fabric_secret = ENV["FABRIC_BUILD_SECRET"]
-    raise "Env var 'FABRIC_BUILD_SECRET' not set" unless fabric_secret
-
-    setup_circle_ci if is_ci
-    sync_code_signing(type: "ad-hoc")
+  desc "Submit a new Beta version to Firebase App Distribution"
+  lane :firebase_beta do |options|
+    app_identifier = options[:app_identifier]
+    raise "Missing parameter app_identifier" unless app_identifier
+    
+    configuration = options[:configuration]
+    raise "Missing parameter configuration" unless configuration
+    
+    firebase_group = options[:firebase_group]
+    raise "Missing parameter firebase_group" unless firebase_group
+    
+    scheme = options[:scheme]
+    raise "Missing parameter scheme" unless scheme
+    
+    firebase_app = ENV["FIREBASE_IOS_APP"]
```

Comment:
Should this come from the options map too so all information is passed to the command the same way? Or is there any specific case this would apply better?

Reply:
@barbosa given that this key gives admin access to our project (e.g. to deploy the app), I believe it fits better as an env var.